blockchain/blocks/src/gossip_block.rs

```diff
@@ -5,15 +5,35 @@ use crate::BlockHeader;
 use cid::Cid;
 use forest_encoding::tuple::*;
 use fvm_ipld_encoding::Cbor;
+use fvm_shared::{address::Address, clock::ChainEpoch};
 
 /// Block message used as serialized `gossipsub` messages for blocks topic.
-#[derive(Clone, Debug, Serialize_tuple, Deserialize_tuple)]
+#[derive(Clone, Debug, PartialEq, Serialize_tuple, Deserialize_tuple)]
 pub struct GossipBlock {
     pub header: BlockHeader,
     pub bls_messages: Vec<Cid>,
     pub secpk_messages: Vec<Cid>,
 }
 
+impl quickcheck::Arbitrary for GossipBlock {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        let header = BlockHeader::builder()
+            .miner_address(Address::new_id(0))
+            .epoch(ChainEpoch::arbitrary(g))
+            .build()
+            .unwrap();
```

Comment:
There is an `Arbitrary` implementation for `BlockHeader`. Can you use it or do you need some custom block headers here?

Reply:
That's a good point, fixed